### ISSUE TITLE
[4.4.0-beta.6] Cannot see assets on field

### ISSUE THREAD

**Issue by 1stevengrant** [craftcms/cms#12820]
### What happened?

When browsing assets, there's a console error on `BaseElementIndex.js` that I reckon is hiding the files in the bucket

![](https://d.pr/i/P5PSDm+)

### Craft CMS version

4.4.0-beta-6

### PHP version

8.1

### Operating system and version

Mac Ventura

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
I’m not able to reproduce this. Can you post the JS error?

**Comment by 1stevengrant** [comment]
```
BaseElementIndex.js:843 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'removeClass')
    at s.constructor._updateSourcePathVisibility (BaseElementIndex.js:843:18)
    at s.constructor.handleResize (BaseElementIndex.js:389:14)
    at s.constructor.init (BaseElementIndex.js:350:12)
    at s.constructor.init (AssetIndex.js:29:12)
    at s.constructor.e [as init] (Base.js:76:36)
    at s.constructor.constructor (Base.js:20:15)
    at s.constructor (Base.js:32:21)
    at s.constructor (Base.js:32:21)
    at new s.constructor (Base.js:32:21)
    at Object.createElementIndex (Craft.js:1968:12)
```

**Comment by brandonkelly** [comment]
Thanks! I was able to figure out how to reproduce it from that, and got this fixed for the official 3.8.0 and 4.4.0 releases (which are now tagged) 🎉
